app/models/shipit/task.rb

```diff
@@ -171,9 +171,9 @@ def spec
       @spec ||= DeploySpec::FileSystem.new(working_directory, stack.environment)
     end
 
-    def enqueue
+    def enqueue(skip_queue: false)
```

Comment:
perhaps `inline` vs `skip_queue` to highlight the fact that it doesn't just run immediately, but runs synchronously?

Reply:
Or even a new method - `run_now!` or `run_inline!` maybe.

Reply:
I'll go with `run_now!` as it reflects more what is going on 👍